Source/WebCore/platform/network/mac/UTIUtilities.mm

```diff
@@ -26,6 +26,7 @@
 #import "config.h"
 #import "UTIUtilities.h"
 
+#import <wtf/HashSet.h>
```

Comment:
Is this needed?

Reply:
Unified sources fix (and like... the file uses HashSet, so yes, it certainly should include it).